.gitlab-ci.yml

```diff
@@ -233,6 +233,17 @@ windows32:
   variables:
     ARCH: "32"
 
+nixos:
+  image: nixorg/nix:latest # Minimal NixOS image which doesn't even contain git
```

Comment:
This seems like a non-reproducible recipe.

Reply:
I know who has built this image but I don't see why is this relevant here. My question is still the same, this seems like a non-reproducible setup, doesn't it?

Reply:
I'm unsure what exactly you are asking? Is this because of `latest`? What can I do to make it reproducible? Use a hash instead?

Reply:
From my part I can live with it if it is hard to improve, even if there are some components that worry such as openssl or git. WDYT @SkySkimmer ?